.github/workflows/e2e.yaml

```diff
@@ -88,34 +88,3 @@ jobs:
         with:
           name: "cere_dev_runtime.compact.compressed.${{ env.GITHUB_SHA }}.wasm"
           path: "./${{ env.CERE_DEV_RUNTIME }}"
-
-  trigger-e2e-tests:
```

Comment:
Let's confirm from @tabakopavel do we need to keep this section or not for the `staging` and `master` branches. 
Initially, we intended to disable the e2e test for the `dev` branch, as the Devnet is an experimental environment that we can break and recover. But Testnet (and probably Qanet) environments should execute e2e tests to ensure compatibility with DDC.